scripts/manual-release.sh

```diff
@@ -40,13 +41,13 @@ else
   EXTRAVERSION_NUMBER=0
 fi
 
-RELEASE="$MAJOR.$MINOR.$PATCH$EXTRAVERSION"
-PRELEASE="$MAJOR.$MINOR.$PATCH.$EXTRAVERSION_NUMBER"
+RELEASE="$MAJOR.$MINOR.$PATCH.$TWEAK$EXTRAVERSION"
+PRELEASE="$MAJOR.$MINOR.$PATCH.$TWEAK"
```

Comment:
I don't think we can use the tweak version, unfortunately, because the least-significant byte of the 32-bit version number is used by the release candidate number (which is 0 for final releases)